scripts/release/publish.sh

```diff
@@ -34,16 +34,26 @@ if [[ "${CI:-}" == "" ]]; then
 fi
 
 version=""
+release_notes=""
+draft=0
 dry_run=0
 
-args="$(getopt -o "" -l version:,dry-run -- "$@")"
+args="$(getopt -o "" -l version:,release-notes:,draft,dry-run -- "$@")"
```

Comment:
For long values like release notes, to avoid formatting problems I think we should take `--release-notes-file` instead of `--release-notes` and use a file like I suggested previously.

Reply:
Done. 👍🏻